src/tile.cpp

```diff
@@ -83,12 +83,13 @@ std::tuple<int, int> Tile::getGoogle()
 std::string Tile::getQuadTree()
 {
     int tmsX = tmsX_;
-    int tmsY = (1 << (zoom_ - 1)) - tmsY_;
+    int tmsY = (1 << zoom_) - 1 - tmsY_;
 
     std::stringstream strStream;
-    generate_n(std::ostream_iterator<int>{strStream}, zoom_, [&, i = zoom_]() mutable {
+   
+    for(int i = zoom_; i > 0; i--){
```

Comment:
Is there a reason to replace `generate_n` with this for loop?

Reply:
```suggestion
    generate_n(std::ostream_iterator<int>{strStream}, zoom_, [&, i = zoom_]() mutable {
        auto digit = 0;
        auto maks = 1 << (i -= 1);
        if ((tmsX & maks) != 0)
        {
            digit += 1;
        }
        if ((tmsY & maks) != 0)
        {
            digit += 2;
        }
        return digit;
    });
    return strStream.str();
```

Also this is ok. I changed it before i have found the real bug:
```
if( digit > 2 ){ digit %= 2; }
```
and
```
 int tmsY = (1 << zoom_- 1) - tmsY_;
```

changed in:
```
 int tmsY = (1 << zoom_) - 1 - tmsY_;
```
I wrote the for loop because i used the pyGeoTile as comparison.